plugins/ruby-foreman-openscap/debian/postinst

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
```

Comment:
depending on what this plugin is bringing (its own assets and/or API extension), postinst and rules need to get modified, see e.g. foreman_salt